nilearn/plotting/find_cuts.py

```diff
@@ -196,6 +196,10 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     orig_data = np.abs(img.get_data())
```

Comment:
Not related to this pr, but for n_cuts I would take a default value of,say, 5. 12 is huge.